Sources/SwiftOracle/BindVar.swift

```diff
@@ -18,7 +18,7 @@ public class BindVar: StringLiteralConvertible, IntegerLiteralConvertible, Boole
         
     }
     public init (_ value: String) {
-        var v = Array(value.nulTerminatedUTF8).map( {Int8(bitPattern: $0) })
+        var v = Array(value.utf8CString)
```

Comment:
in addition to this:
lip@ubuntu:~/for_orcl_driver$ swift build -Xlinker -L$USER_LIBS/lib/
Compile cocilib empty.c
Linking cocilib
Compile Swift Module 'SwiftOracle' (5 sources)
/home/lip/for_orcl_driver/Packages/SwiftOracle-0.1.5/Sources/SwiftOracle/Connection.swift:114:19: warning: value 'connection' was defined but never used; consider replacing with boolean test
        guard let connection = connection else {
              ~~~~^~~~~~~~~~~~~
                                          != nil
Compile Swift Module 'for_orcl_driver' (1 sources)
Linking ./.build/debug/for_orcl_driver